.github/workflows/lints.yml

```diff
@@ -0,0 +1,17 @@
+name: Lints
+
+on: [pull_request, push]
```

Comment:
Do we really need to run this on pushes? In which scenario something would be pushed to a branch that is not connected to a pull request?

Reply:
@davialexandre  this would help for development. For example, on feature branch if developer push the code, we can ensure that the action can be executed flawlessly before creating a pull request.